en/docs/develop/writing-a-custom-event-handler.md

```diff
@@ -1,228 +1,161 @@
-                  serviceProvider.setDescription(applicationDescription);
-                  applicationMgtService.createApplication(serviceProvider, tenantDoamin, username);
-              } catch (Exception e) {
-                  throw new WorkflowException(e.getMessage(), e);
-              }
-          } else {
-              if (retryNeedAtCallback()) {
-                  unsetWorkFlowCompleted();
-              }
-              if (log.isDebugEnabled()) {
-                  log.debug("Adding user is aborted for SP '" + applicationName + "', Reason: Workflow response was " +
-                                  status);
-              }
-          }
-      }
-
-      @Override
-      public boolean retryNeedAtCallback() {
-          return true;
-      }
-
-     @Override
-     public String getEventId() {
-          return "ADD_SP";
-      }
-     
-     @Override
-     public Map<String, String> getParamDefinitions() {
-          return PARAM_DEFINITION;
-      }
-
-      @Override
-     public String getFriendlyName() {
-          return "Add SP";
-      }
-
-      @Override
-     public String getDescription() {
-          return "";
-      }
-
-      @Override
-     public String getCategory() {
-          return "SP Operations";
-      }
-
-      public boolean startSPCreateWorkflow (ServiceProvider serviceProvider, String tenantDomain, String userName)
-              throws WorkflowException{
-          Map<String, Object> wfParams = new HashMap<>();
-          Map<String, Object> nonWfParams = new HashMap<>();
-          wfParams.put("Application ID",serviceProvider.getApplicationID());
-          wfParams.put("Application Name",serviceProvider.getApplicationName());
-          wfParams.put("Application Description",serviceProvider.getDescription());
-          wfParams.put("Tenant Domain",tenantDomain);
-          wfParams.put("Username",userName);
-          String uuid = UUID.randomUUID().toString();
-          Entity[] entities = new Entity[1];
-          entities[0] = new Entity(serviceProvider.getApplicationName(), "SP", -1234);
-          if (!Boolean.TRUE.equals(getWorkFlowCompleted()) && !isValidOperation(entities)) {
-              throw new WorkflowException("Operation is not valid.");
-          }
-          boolean state = startWorkFlow(wfParams, nonWfParams, uuid).getExecutorResultState().state();
-          return state;
-      }
-
-      @Override
-      public boolean isValidOperation(Entity[] entities) throws WorkflowException {
-          //Check if the operation is valid, eg:- Is there a SP already added and not approved with the same name as
-          // this SP.
-          return true;
-      }
-    }
+The WSO2 Identity Server eventing framework can be used to trigger some events such as user operations. The eventing framework also supports handlers which can be used to do some operations based on the published events.
```

Comment:
**....can be used to trigger some events such as user operations.**
Shall we add a couple of sample events, and mention that full list is available below. ex: `can be used to trigger events such as user operation events like PRE_SET_USER_CLAIMS, POST_ADD_USER`

**...framework also supports handlers which can be used to do some operations based on the published events**
Shall we add an example operation as well? ex; `which can be used to do operations upon a triggered event. For example, validate the changed user password against  the previously used entries, upon a PRE_UPDATE_CREDENTIAL event`